readme.md

```diff
@@ -159,7 +159,7 @@ Jira notifications are messages sent to a Mattermost channel when a particular e
 
 Notifications and webhooks can be used together or you can opt for one of them.
 
-![This is a channel notification of a new bug that was created in Jira](https://github.com/mattermost/mattermost-plugin-jira/assets/74422101/e7020c3e-48f6-4825-8193-6a189f6c96eb)
+![This is a channel notification of a new bug that was created in Jira](https://github.com/mattermost/mattermost-plugin-jira/assets/74422101/e15de4fe-1cb3-47d1-9b0d-538ab82ec91d)
```

Comment:
I'm curious what's going on with this long hash here. @chriswachira Do you know how this URL is generated?

I wonder if this would also work:

```suggestion
![This is a channel notification of a new bug that was created in Jira](./assets/attach-from-post.png)
```

Reply:
Hello and Happy New Year!

Uploading the image as an attachment on a PR description (even without posting the PR) generates such a URL when you select "Copy link address" from the browser's menu.

Yes, your suggestion also works.

Reply:
@chriswachira Happy New Year! Hope you're doing well

On the subject of how to reference to the image, I think we would prefer the `./assets/attach-from-post.png` approach, as it's easier to trace where the image is coming from. I applied the suggestion above, but the image is not shown properly in the readme, so I've reverted that change. The current change seems to not render the image properly either, when I look at the "rich diff" in the Files Changed view. @chriswachira Do you know why this might be the case?

![CleanShot 2024-01-04 at 12 37 05](https://github.com/mattermost/mattermost-plugin-jira/assets/6913320/dbd7495d-41b0-46c7-855e-bc1edded9aaf)

Reply:
I've applied the relative path suggestion and it seems to be okay

Reply:
Hi @mickmister! Kindly let me know if I should proceed to squash the commits for a final review.